### ISSUE TITLE
Can't use movePluginBeforeInitAuthPlugin when server-side framework is Koa

### ISSUE THREAD

**Issue by Ricklin90085** [nuxt-community/firebase-module#175]
### Version
@nuxtjs/firebase: ^5.0.7
firebase: ^7.14.1
nuxt: 2.12.2

### Steps to reproduce
1. `create-nuxt-app` and server-side framework use `Koa`
2. `npm i firebase @nuxtjs/firebase --save`
3.  follow
https://firebase.nuxtjs.org/guide/helpers/#movepluginbeforeinitauthplugin

### What is Expected?
Support `Koa` to use `movePluginBeforeInitAuthPlugin`.

### What is actually happening?
When I add `import { movePluginBeforeInitAuthPlugin } from '@nuxtjs/firebase/lib/helpers'` in my `nuxt.config.js` and run `npm run dev`, npm will throw an error:
![Koa](https://imgur.com/xEweKNu.jpg)
Later I found the reason maybe is `Koa` does not suppert `ES6's module system (import/export)` .
From the [source code](https://github.com/nuxt-community/firebase-module/blob/dev/lib/helpers/index.js#L6), It is now only supported `ES6's module system (import/export)`.
So I think maybe support `CommonJS` is a good way to support `Koa`？or there are other better ways ?
Thank you for helping :)

**Comment by lupas** [comment]
Hey @Ricklin90085 

Thanks for the report.
Yep, that makes sense. I'll look into it and try to change that to CommoJS in the next update.

This might not work at all, not sure, but maayyybeeee the [`build.transpile`](https://nuxtjs.org/api/configuration-build#transpile) option of nuxt could help you here too:
```js
build: {
    transpile: ['@nuxtjs/firebase']
   // or
   transpile: ['@nuxtjs/firebase/lib/helpers']
  }
```

Could you try that out and let me know if it works? Otherwise, just copy & paste [this code](https://github.com/nuxt-community/firebase-module/blob/dev/lib/helpers/index.js#L6) somewhere and use it as a CommonJS module until we fix this.

**Comment by Ricklin90085** [comment]
@lupas 
I try the [build.transpile](https://nuxtjs.org/api/configuration-build#transpile) option, but unfortunately it doesn't work, I think maybe because `import` is on the first line, so it crash at the beginning.
The good news is copy & paste `helpers/index.js` and use it as a CommonJS module is helpful to me🎉
Thanks again for your help👍

**Comment by lupas** [comment]
@Ricklin90085 
Glad that did the trick. Yeah I guess the transpile only works for modules/plugins imported through nuxt, not general imports, which this function basically is. 

I'll keep this issue open until the CommonJS change is done and deployed.

**Comment by lupas** [comment]
https://github.com/nuxt-community/firebase-module/commit/806ecce0a3b85478ef71a383e6a64c018c1b76e2

**Comment by lupas** [comment]
Should be fixed with [v5.2.0]( https://github.com/nuxt-community/firebase-module/releases/tag/v5.2.0)
